ui/00_message/tutorial_guide/tutorial_guide.xml

```diff
@@ -669,7 +685,8 @@
   </message>
   <message>
     <key>TUTORIAL_275</key>
-    <original>侵食魔について：大型敵 </original>
+    <original>侵食魔について：大型敵</original>
```

Comment:
I know sorry, dunno how that happened, I probably erased it by mistake and forgot about the space at the end.